timemachine/fe/free_energy.py

```diff
@@ -54,13 +53,16 @@
 )
 
 
+# (YTZ): make dataclass/immutable later?
+# (YTZ): deduplicate with rbfe.py
 class HostConfig:
-    def __init__(self, omm_system, conf, box, num_water_atoms, omm_topology):
-        self.omm_system = omm_system
+    def __init__(self, host_system, conf, box, num_water_atoms, omm_topology, masses):
```

Comment:
@mcwitt  - I should probably add type annotations to this class but it will probably blow up the PR considerably. Okay if I defer?